schema/queue/message/impl/RabbitMqMessageRouter.go

```diff
@@ -52,41 +51,45 @@ func (cmr *CommonMessageRouter) Close() {
 func (cmr *CommonMessageRouter) SendAll(msgBatch *p_buff.MessageGroupBatch, attributes ...string) error {
 	attrs := MQcommon.GetSendAttributes(attributes)
 	pinsFoundByAttrs := cmr.connManager.QConfig.FindQueuesByAttr(attrs)
-	if len(pinsFoundByAttrs) != 0 {
-		for pin, config := range pinsFoundByAttrs {
-			cmr.Logger.Debug().Msgf("Try filtering for pin: %v", pin)
-			if cmr.filterStrategy.Verify(msgBatch, config.Filters) {
-				if e := log.Debug(); e.Enabled() {
-					e.Msgf("Message batch with first id: %v  matched pin: %v filter", defaultStrategy.ExtractID(msgBatch), pin)
-				}
-				sender := cmr.getSender(pin)
-				err := sender.Send(msgBatch)
-				if err != nil {
-					cmr.Logger.Fatal().Err(err).Send()
-					return err
-				}
-				if e := log.Debug(); e.Enabled() {
-					e.Msgf("Message batch with first id: %v was sent to pin: %v ", defaultStrategy.ExtractID(msgBatch), pin)
-				}
-			} else {
-				if e := log.Debug(); e.Enabled() {
-					e.Msgf("Message batch with first id: %v didn't match pin: %v filter", defaultStrategy.ExtractID(msgBatch), pin)
-				}
+	if len(pinsFoundByAttrs) == 0 {
+		cmr.Logger.Fatal().Msg("No such queue to send message")
```

Comment:
Please, do not call `Fatal` on the logger. It will call `panic` under the hood. If this is an unacceptable case we should rather return an error instead of panicing